src/scss/placeholders/_host.scss

```diff
@@ -0,0 +1,12 @@
+%component-host {
+  /* Base ":host" styles for the component */
+  box-sizing: border-box;
+  * {
+    box-sizing: border-box;
+  }
+  color: var(--calcite-app-foreground);
+  font-family: var(--calcite-app-font-family);
```

Comment:
adding the full font stack to every component seems like a waste. Could just use `inherit`

Reply:
Yeah, I think we can work with @asangma to clean these up and get them in sync with `calcite-components`